### PR TITLE
extend tests to cover the insensitive analysis

### PR DESCRIPTION
**Pull request by nikomatsakis**
r? @lqd

### CODE REVIEW

File: src/test.rs [hunk 1]
```diff
@@ -12,6 +12,15 @@ use std::path::Path;
 
 fn test_facts(all_facts: &AllFacts, algorithms: &[Algorithm]) {
     let naive = Output::compute(all_facts, Algorithm::Naive, true);
+
+    // If the insensitive analysis concludes no errors, then naive
+    // should also.
+    let insensitive = Output::compute(all_facts, Algorithm::LocationInsensitive, false);
+    if insensitive.errors.is_empty() {
+        assert_equal(&naive.errors, &insensitive.errors);
```
Comment (nikomatsakis) [inline src/test.rs]: maybe we could strengthen this to a subset test

Comment (lqd) [reply src/test.rs]: it could be interesting indeed

**Comment by lqd** [comment]
(nice of github to show the comment more than once)

**Comment by nikomatsakis** [comment]
Rebased, added subset.

**Pull request merged**